packages/SwingSet/src/devices/plugin-src.js

```diff
@@ -30,29 +30,32 @@ export function buildRootDeviceNode(tools) {
   // Register our first state.
   saveState();
 
+  function register(mod, index) {
```

Comment:
I don't really understand the plugin device, so I'll defer to @michaelfig 's review for this part.